charts/seed-bootstrap/templates/gardener-seed-admission-controller/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: gardener-seed-admission-controller
+  labels:
+    app: gardener
+    role: seed-admission-controller
+webhooks:
+  - name: seed.admission.core.gardener.cloud
+    rules:
+    - apiGroups:
+      - apiextensions.k8s.io
+      apiVersions:
+      - v1alpha1
+      - v1beta1
+      - v1
+      operations:
+      - DELETE
+      resources:
+      - customresourcedefinitions
```

Comment:
Why don't creat an additional rule with `objectSelector` selecting `gardener.cloud/deletion-protected: "true"`

Reply:
Object selectors only work with Kubernetes 1.15+. Our minimum seed Kubernetes version is 1.11, though. Hence, we can not use it yet.

Reply:
Well for `<1.15` it'll send all objects as before, but it'll filter them out in `1.15+`.

Reply:
OK, I can add it. Though, it doesn't allow us to remove our own label-filtering logic. I'll add a TODO there that this logic can be removed once the minimum seed version is bumped to >= 1.15.